ci/snap/deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+set -o pipefail
+
+# not a tagged release, abort
+[[ "$TRAVIS_TAG" != "$TRAVIS_BRANCH" ]] && exit 0
+
+openssl enc \
+  -aes-256-cbc \
+  -md sha512 \
+  -pbkdf2 \
+  -iter 1000 \
+  -a -d \
+  -in .snapcraft/travis_snapcraft.cfg \
+  -out .snapcraft/snapcraft.cfg -k $SNAP_SECRECT_KEY
```

Comment:
why doesn't `snapcraft` do this for us? Weird that we have to maintain a bunch of scripts and `openssl` invocations, to support snap.

Reply:
This is pretty standard for travis and deployments that depend on credentials. It is not isolated to snapcraft. This is one of the many ways to keeps secrets in travis. I felt this would put the least burden on the maintainers, as they only need to commit the secret key to travis.